apps/test/unit/templates/manageStudents/ManageStudentsAgeCellTest.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {shallow, mount} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import sinon from 'sinon';
+import {UnconnectedManageStudentAgeCell as ManageStudentAgeCell} from '@cdo/apps/templates/manageStudents/ManageStudentsAgeCell';
+
+
+describe('ManageStudentAgeCell', () => {
+  const DEFAULT_PROPS = {
+    id: 2,
+    isEditing: false,
+    editedValue: 13,
+  };
+
+  let editStudent, setSharingDefault;
+
+  beforeEach(() => {
+    editStudent = sinon.spy();
+    setSharingDefault = sinon.spy();
+  });
+
+  it('renders the age set for the student, when not editing', () => {
+    const wrapper = shallow(
+      <ManageStudentAgeCell
+        {...DEFAULT_PROPS}
+        age={10}
+        editStudent={editStudent}
```

Comment:
These first two tests don't actually need spies, so it's a bit confusing to use them. The beforeEach() should be nested to only refer to the last two tests.

Reply:
All of the tests need the `editStudent` and `setSharingDefault` functions because they are required props for `ManageStudentAgeCell`.  Since those functions are no longer being supplied by Redux, are you suggesting I just pass in dummy functions,  something like`{() => {console.log("fakeEditAll was called");}}` for the first 2 tests?

Reply:
I was, but I guess spying is making the same thing, so actually I think it's fine to leave it.